### ISSUE TITLE
Issue Report: Not compiling any .scss files in VS Code

### ISSUE THREAD

**Issue by brianheath** [glenn2223/vscode-live-sass-compiler#322]
### UNEXPECTED ERROR

**Machine & Versions**
| Item | Value |
|----------------------:|:-----------------------|
| Win 10 Pro | v22H2 |
| VS Code | v1.81.1 |
| Platform | win32 x64 |
| Node | 16.17.1 (110) |
| Live Sass | 6.1.1 |
<details><summary>Installed Extensions</summary><div>
- vscode.css-language-features (1.0.0)<br/>- vscode.debug-auto-launch (1.0.0)<br/>- vscode.emmet (1.0.0)<br/>- vscode.git (1.0.0)<br/>- vscode.git-base (1.0.0)<br/>- vscode.github (0.0.1)<br/>- vscode.github-authentication (0.0.2)<br/>- vscode.html-language-features (1.0.0)<br/>- vscode.json-language-features (1.0.0)<br/>- vscode.merge-conflict (1.0.0)<br/>- vscode.microsoft-authentication (0.0.1)<br/>- ms-vscode-remote.remote-wsl-recommender (0.0.19)<br/>- vscode.typescript-language-features (1.0.0)<br/>- eamodio.gitlens (14.2.1)<br/>- glenn2223.live-sass (6.1.1)<br/>- HookyQR.beautify (1.5.0)<br/>- olback.es6-css-minify (3.3.3)
</div></details>

**LOG**: 
```JSON
{
"NO LOG": "PLEASE SPECIFY YOUR ISSUE BELOW"
}
```
=======================
### Issue:
Nothing is compiled when updating & saving any `*.scss` partials or main files in VS Code.
Output messages claim the files are excluded in settings, but there are no exclusions:
`The file has not been compiled as it's excluded by user settings`

### Steps taken:
- Nothing has changed since (Friday?) when it _was_ working last (AFAIK)
- Rebooted computer
- Disabled/Re-enabled extension
- Uninstalled/Reinstalled extension
- Changed/removed/added settings in `settings.json` file
  - Removed `excludeList` as well as all other `liveSassCompile` settings 
  - Setting `compileOnWatch` to true does not produce any results
- Changed to a different workspace
- Attempted on multiple files
- Disabled all other extensions
- Added `includeItems` after the `excludeList` to no avail:
```JSON
    "liveSassCompile.settings.includeItems": [
        "/**/scss/*.sass",
        "/**/scss/*.scss"
    ],
```

### Details:
- Windows 10 Pro 22H2

**When saving:**
```
VS Code event: "onDidSaveTextDocument"
--------------------
Found the workspace folder
Workspace Name: Home
--------------------
SASS file saved
A SASS file has been saved, starting checks
--------------------
Checking SASS path isn't excluded
Path: \\titan\inetSource\wwwdev\Home\css\scss\_become-a-dealer.scss
--------------------
No base folder override found. Keeping workspace folder
--------------------
Searching folder
Search results
--------------------
File not found, must be excluded
--------------------
File excluded
The file has not been compiled as it's excluded by user settings
Path: \\titan\inetSource\wwwdev\Home\css\scss\_become-a-dealer.scss
--------------------
```

**Command called: "liveSass.command.debugInclusion"**
```
--------------------
Checking current file
\\titan\inetSource\wwwdev\Home\css\scss\_become-a-dealer.scss
--------------------
Found the workspace folder
Workspace Name: Home
--------------------
Checking SASS path isn't excluded
Path: \\titan\inetSource\wwwdev\Home\css\scss\_become-a-dealer.scss
--------------------
No base folder override found. Keeping workspace folder
--------------------
Searching folder
Search results
--------------------
File not found, must be excluded
--------------------
File excluded
The file is excluded based on your settings, please check your configuration
--------------------
```

**Command called: "liveSass.command.debugFileList"**
```
--------------------
Getting SASS files
Query pattern: undefined
Can be overwritten: true
--------------------
Checking folder 1 of 1
Folder: Home
--------------------
No base folder override found. Keeping workspace folder
--------------------
Found 0 SASS files
--------------------
Getting SASS files
Query pattern: /**/_*.s[ac]ss
Can be overwritten: false
--------------------
Checking folder 1 of 1
Folder: Home
--------------------
No base folder override found. Keeping workspace folder
--------------------
Found 0 SASS files
--------------------
Getting SASS files
Query pattern: /**/node_modules/**,/**/.vscode/**,/**/includes/**
Can be overwritten: false
--------------------
Checking folder 1 of 1
Folder: Home
--------------------
No base folder override found. Keeping workspace folder
--------------------
Found 0 SASS files
--------------------
Extension Info
--------------------
Current File
--------------------
\\titan\inetSource\wwwdev\Home\css\scss\_become-a-dealer.scss
--------------------
Workspace Folders
--------------------
[0] Home
\\titan\inetSource\wwwdev\Home
--------------------
Checking workspace folder 1 of 1
Path: \\titan\inetSource\wwwdev\Home
--------------------
--------------------
Current Include/Exclude Settings
--------------------
Include: [ NULL ]
Exclude: [ /**/node_modules/**, /**/.vscode/**, /**/includes/** ]
--------------------
Included SASS Files
--------------------
--------------------
Included Partial SASS Files
--------------------
--------------------
Excluded SASS Files
--------------------
--------------------
```

### settings.json
```JSON
{
    "terminal.integrated.defaultProfile.windows": "Git Bash",
    "liveSassCompile.settings.autoprefix": [
        "> 1%",
        "last 2 versions"
    ],
    "liveSassCompile.settings.excludeList": [
        "/**/node_modules/**",
        "/**/.vscode/**",
        "/**/includes/**"
    ],
    "liveSassCompile.settings.formats": [
        {
            "format": "compressed",
            "extensionName": ".min.css",
            "savePath": "~/.."
        }
    ],
    "explorer.confirmDelete": false,
    "security.workspace.trust.untrustedFiles": "open",
    "liquid.format": true,
    "workbench.startupEditor": "none",
    "diffEditor.ignoreTrimWhitespace": false,
    "liveSassCompile.settings.showOutputWindowOn": "Trace",
    "liveSassCompile.settings.useNewCompiler": true,
    "files.associations": {
        "*.asp": "html"
    },
    "git.openRepositoryInParentFolders": "always",
    "security.allowedUNCHosts": [
        "titan"
    ],
    "liveSassCompile.settings.compileOnWatch": false,
}
```

**Comment by glenn2223** [comment]
Sorry for the delay. 

Yeah, very strange! Like you said there wasn't a release on the Friday.

Some thoughts:
- rollback to the one of the previous versions, in case the extension update was delayed (for some reason)
- some users have reported issues when working on network drives - one report ([here](https://github.com/glenn2223/vscode-live-sass-compiler/issues/258#issuecomment-1422160720)) said that shadow files were the underlying issue. 
- I assume the compilation works fine when the files are on the local machine??
- if it's a [remote development](https://code.visualstudio.com/docs/remote/remote-overview) situation, I assume you have the extension installed on the server rather than the local machine?

**Comment by brianheath** [comment]
Well now THAT'S interesting. This is local development but it's on the network using UNC. What's interesting is that when I map the network path to an actual drive letter, **it works**. But when I go back to UNC and edit the same exact file, I get that error.

Yes, it works perfectly on a local drive.

The other very odd thing about it is, I have been using UNC for 4+ years doing development here and have never had a problem (mapping a drive letter breaks our Git repo) until I wrote this issue report.

I am at a total complete loss as to why this happened all of a sudden. None of the paths have changed, and nobody uses a Mac here so there aren't any shadow or hidden files created as a result.

I also went into Git BASH and ran sass manually using the same exact UNC path and it worked just fine.

Is there any kind of setting that I can add or change in this case, to perhaps tell it to look at this complete pathway? Should I be using double back/forward slashes or anything? I'm so utterly clueless as to what to do at this point.

Thank you so much for your help.


P.S. This has been the greatest extension EVER, and I will be forever singing its praise regardless of how my situation works out!

**Comment by brianheath** [comment]
I absolutely would be willing to test that - whatever I can do to help!

I actually did not roll back the extension itself, but I will do that to see if that changes anything.

Come to think of it now... There _was_ something that I noticed at the time:
There was a change that made me prefix the patterns with "/" in the file exclusions & inclusions. That seems to have coincided with all of this. That makes me think you were right about the delayed update. I will do a rollback in a bit and let you know the outcome.

Thank you!
